### ISSUE TITLE
Loading Schematic doesn't work on Multiplayer Server.

### ISSUE THREAD

**Issue by Definition** [Lunatrius/Schematica#444]
INFO:

Server Version: Pixelmon Reforged 7.0.9 (Minecraft 1.12.2)
Minecraft: 1.12.2
Forge: 1.12.2-forge.1.12.2-14.23.5.2768
Schematica: Schematica-1.12.2-1.8.0.169-universal
Lunatrius Core: LunatriusCore-1.12.2-1.2.0.42-universal

ISSUE:

I've created an elaborate building in singleplayer using different plugins (WorldEdit, Voxel, etc) and was able to copy AND reprint using schematica in a singleplayer world. When looking to transfer it over to a survival multiplayer server, I can't even get the Schematic to load up and show on my screen. Nothing on schematica works except for the GUIs popping up. It seems as if almost everything has been disabled pretty much.

Here is a video of the issue at hand: https://streamable.com/ui7b2

Schematica doesn't seem to run on any online server as far as I can tell. It's only working in singleplayer.

**Comment by pHpoo** [comment]
yeah you clearly have it installed wrong then cause i've been using schematica for ages in online server, most current time... right now as i type this lol. you might need to go in and look at the settings and see if there different because of certain pluggins the server is running which might conflict with your button setup
